cmd/meradm/server.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"errors"
+
+	"strconv"
+
+	"fmt"
+
+	"strings"
+
+	"github.com/golang/protobuf/ptypes/wrappers"
+	"github.com/sky-uk/merlin/types"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var serverCmd = &cobra.Command{
+	Use:   "server [add|edit|del]",
+	Short: "Modify a real server",
+}
+
+func validServiceIDHostPort(_ *cobra.Command, args []string) error {
+	if len(args) != 2 {
+		return errors.New("requires two arguments")
+	}
+	b := []byte(args[1])
+	if !hostPortRegex.Match(b) {
+		return errors.New("must be ip:port")
```

Comment:
Yep all it's checking for is the colon exists. That is the minimum meradm needs to pull out the IP and port. I want to avoid duplicating validation we can do on the server side if possible (like doing a parse of the IP and so on).